000-config.R

```diff
@@ -1,78 +1,83 @@
 ### Workflow Configuration Settings ###
 
-# Check that we are in the correct working directory
-if(!basename(here::here(getwd())) == 'downscaling') {
-  PEcAn.logger::logger.error("Please run this script from the 'downscaling' directory")
-}
-
-## Global configuration settings for logging
+parser <- argparse::ArgumentParser()
 
-options(
-  # Print all tibble columns
-  tibble.width = Inf,
-  # Suppress readr::read_csv messages
-  readr.show_col_types = FALSE
+## Set development vs production mode ##
+# Dev mode speeds up workflows by subsetting data for testing and debugging
+parser$add_argument("--production",
+  type = "logical", default = TRUE,
+  help = "Set to true for production mode, false for faster development (default: FALSE)"
```

Comment:
default/doc mismatch?
```suggestion
  type = "logical", default = TRUE,
  help = "Set to true for production mode, false for faster development (default: TRUE)"
```